drivers/net/bnxt/bnxt_ethdev.c

```diff
@@ -793,6 +794,78 @@ static int bnxt_rss_hash_update_op(struct rte_eth_dev *eth_dev,
 	return 0;
 }
 
+int rte_pmd_bnxt_set_tx_loopback(uint8_t port, uint8_t on)
```

Comment:
VFd need this function to enable/disable loopback mode and it looks that loopback switch not implemented in HWRM.